### PR TITLE
MockInventoryDriver.py QuadsNativeInventoryDriver.py functions

### PR DESCRIPTION
**Pull request by vsemp**
Added init_data sync_data write_data load_data. Fucntions are the same for Mock and QuadsNative. The fucntions are just copies from libquads.py.

### CODE REVIEW

File: lib/hardware_services/inventory_drivers/hil.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+url: http://127.0.0.1:5000    # the address of HIL server
```
Comment (saacg) [inline lib/hardware_services/inventory_drivers/hil.yml:1]: @vsemp see #54 
as we discussed prior, I added this as a parameter to conf/quads.yml

File: lib/hardware_services/network_drivers/hilapi.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    return do_get(url)
+
+def show_network(network):
+    url = make_url('network', network)
+    return do_get(url)
+
+def show_node(node):
+    url = make_url('node', node)
+    return do_get(url)
+
```
Comment (saacg) [inline lib/hardware_services/network_drivers/hilapi.py:107]: @vsemp see #54

File: lib/hardware_services/inventory_drivers/hilapi.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    return do_get(url)
+
+def show_network(network):
+    url = make_url('network', network)
+    return do_get(url)
+
+def show_node(node):
+    url = make_url('node', node)
+    return do_get(url)
+
```
Comment (saacg) [inline lib/hardware_services/inventory_drivers/hilapi.py:107]: @vsemp see #54

**Comment by saacg** [comment]
@vsemp did you test your changes before making this pr? I just cloned the branch and have not been able to get any commands to work without throwing errors when using QuadsNative or Mock as the hardware service.

File: lib/hardware_services/inventory_drivers/hil.yml (hunk 1 above)
Comment (vsemp) [reply lib/hardware_services/inventory_drivers/hil.yml:1]: Yes. We discussed that. And I agreed with you. But I don't see it added.

So can we merge this first and then we can move this line anywhere?

Comment (saacg) [reply lib/hardware_services/inventory_drivers/hil.yml:1]: https://github.com/BUEC528-HIL-QUADS/quads/pull/54/files#diff-54b53b5e855e4de6cd714db5a7e6a638

**Comment by portante** [comment]
@vsemp, when you rebase these changes on top of `class/master`, please be sure to setup your git Identity, as described at [Git - First Time Git Setup](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup).  Most of your changes are committed with their authorship of the system administrator.

Also, please rebase onto `class/master` following the procedure detailed in the document I created, entitled "[Git Rebase Workflow](https://docs.google.com/document/d/1aXyzKKeaSDakrM0MmyV0mKw-6QWvbTouezYX8VsKK_Q/edit?usp=sharing)".

**Pull request closed without merging**